### PR TITLE
Beta 13

### PR DESCRIPTION
**Pull request by dfvalero**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@zopauk/react-components",
   "sideEffects": false,
   "description": "Shared react styled components for all the Zopa projects.",
-  "version": "4.0.0--beta.12",
+  "version": "4.0.0-beta.13",
```
Comment (moshie) [inline package.json:5]: Should this be `4.0.0--beta.13`?

Comment (dfvalero) [reply package.json:5]: I don't know, in my note I have this command `npm version --newVersion x.0.0-beta.x`

Comment (dfvalero) [reply package.json:5]: also, I released 4.0.0-beta.13

Comment (poteirard) [reply package.json:5]: Actually I don't know why are using this `--beta` thing. I would stick with what `yarn` generates when you run `yarn publish --prerelease`. Which at least it follows a spec

Comment (poteirard) [reply package.json:5]: @lluia what's the reason of following that beta thing?

**Comment by dfvalero** [comment]
Closed because we released beta.14. A PR will come in a sec

**Pull request closed without merging**